procrastinate/worker.py

```diff
@@ -36,6 +36,10 @@ def __init__(
     async def run(self) -> None:
         await self.job_store.listen_for_jobs(queues=self.queues)
 
+        self.logger.debug(
+            f"Log start one queues {self.queues}",
```

Comment:
s/one/on

Reply:
`self.queues` is `None` if the worker listens on all the queues

Reply:
IMO the message could be changed, we don't want a log message that says we are (just) starting to log something :)

Reply:
Note that this is a debug log @mgu :) .

Reply:
I'll change that. In fact, we were not sure about adding queues information, I had to removed some because It was too heavy. (like, two if to know if I can log something, for a debug log ...)